src/c++/perf_analyzer/custom_load_manager.cc

```diff
@@ -112,4 +114,27 @@ CustomLoadManager::GetCustomRequestRate(double* request_rate)
   return cb::Error::Success;
 }
 
+cb::Error
+CustomLoadManager::ReadTimeIntervalsFile(
```

Comment:
This was simply moved from the utils file so I could mock it to remove filesystem functionality from being run during a unit test.